sympy/core/expr.py

```diff
@@ -24,8 +24,8 @@ class Expr(Basic, EvalfMixin):
     used only for argument storage and expression manipulation, i.e.
     pattern matching, substitutions, etc).
 
-    Should us _eval_is_X (_eval_is_gt/_eval_is_lt _eval_is_le) if we want to override
-    the comparison functions.
+    If you want to override the comparisons of expressions:
+    Should use _eval_is_X (where X is gt, ge, lt, or Eq).
```

Comment:
Do you mean `le` or `Eq`?

Reply:
No I don't. We do not check for le, only for less than, and that's only the case when both expressions have, is_extended_real set to true. Less than or equal to, is just fuzzy_not over greater than.

Reply:
Isn't `ge` fuzzy_not over less than? (or were you setting me up to ask this :-))

Reply:
Well, it might be. Sorry, I made a mistake in what I wrote last night. I meant we only check for less than when one or both expressions have is_extended_real set to false. If both number are not extended reals, ge may not be fuzzy_not over less than. See oscarbenjamin's earlier comments.